pallets/msa/README.md

```diff
@@ -0,0 +1,71 @@
+# MSA Pallet
+
+The MSA Pallet provides functionality for handling Message Source Accounts.
+
+## Summary
+
+The Message Source Account (MSA) is the primary user account system on Frequency for Messages and Stateful Storage.
+All users on Frequency must have an MSA in order to:
+
+1. Acquire a User Handle
+2. Delegate tasks to Providers (defining specific tasks for specific providers)
+3. Become a Provider so they may do Provider level tasks on their own behalf
+4. Have Message or Stateful Storage data
+
+### MSA Id and Keys
+
+Once a user creates an MSA, they are assigned an MSA Id–a unique number permanently assigned to the number at the time of creation as well as one or more control keys attached for control.
```

Comment:
```suggestion
Once a user creates an MSA, they are assigned an MSA Id–a unique number permanently assigned to the user at the time of creation as well as one or more control keys attached for control.
```

Not sure if "to the user" is correct, either... maybe just remove the phrase?

Reply:
Updated to:

```
Once a user creates an MSA, they are assigned an MSA Id, a unique number the time of creation with one or more keys attached for control.
```